README.md

```diff
@@ -102,6 +102,18 @@ Alternatively, just compile the `index.ts` using `tsc`.
 > node ./basic-usage/index.js
 ```
 
+## Running in Docker
+If use tea-school in Docker you can pass to Puppeteer arguments.
```

Comment:
According to [this](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md) troubleshooting, `--no-sandbox` isn't required.
Let's not coerce the users to make a discouraged choice, and just give them the option to pass their own `puppeteer` options if they want to.

So let's drop this documentation and keep the puppeteer options you suggested in the `GeneratePdfOptions`